### ISSUE TITLE
Replace static image of table with HTML table

### ISSUE THREAD

**Issue by brentryanjohnson** [ONRR/doi-extractives-data#3067]
The [How it works-->Revenues table](https://revenuedata.doi.gov/how-it-works/revenues/#federal-lands-and-waters) has valuable information, but it's a `png` image with insufficient `alt` text. Consequently, it's content is neither accessible nor machine-readable. Let's replace it with an HTML table.

![static table image](https://user-images.githubusercontent.com/32855580/42849705-46c4cd02-89d9-11e8-89a4-c9da09c8f63a.png)

**Comment by jennmalcolm** [comment]
First step, create a wireframe using current table patterns.

**Comment by mcharg** [comment]
First pass wireframe:
https://z34mw5.axshare.com/#g=1&p=how_revenue_works

@brentryanjohnson Thoughts?
